src/main/scala/format/pgn/Node.scala

```diff
@@ -0,0 +1,91 @@
+package chess
+package format
+package pgn
+
+import cats.syntax.all.*
+import cats.derived.*
+import cats.*
+
+import monocle.Traversal
```

Comment:
new dependency?

Reply:
for Monocle, we don't really need to include it as dependency. It gives us more power to manipulate/query our data structure though. But we can just write those filterKey manually.